### DISCUSSION TITLE
Unable to dump database

### DISCUSSION THREAD

**Discussion by t7m** [dbeaver/dbeaver#34749] [General]
### Description

Right-click my database... tools... dump database.

Task 'MySQL dump' finished at Tue Jul 09 10:47:27 EDT 2024
2024-07-09 10:47:27.868 - IO error: Process failed (exit code = 2). See error log.

Why? And where is the error log?

### DBeaver Version

Community Version 24.1.2.202407071610

### Operating System

Windows 10

### Database and driver

1. MySql

### Steps to reproduce

See description.

**Comment by ShadelessFox** [comment]
Hello @t7m,

What database version is this? What driver are you using? Both can be checked by performing the "test connection" operation.

**Comment by t7m** [comment]
Server: MySQL 5.7.12-log
Driver: MySQL Connector/J mysql-connector-j-8.2.0 (Revision: 06a1f724497fd81c6a659131fda822c9e5085b6c)

**Comment by ShadelessFox** [comment]
Thanks. What native client did you use? Perhaps you're using tools that are too modern or old and don't work with your particular database version.

![image](https://github.com/dbeaver/dbeaver/assets/35821147/09426733-b4a2-4777-b596-e9c7b41ad801)

**Comment by t7m** [comment]
MySQL Workbench ended up working for me. I have no need for DBeaver now. 

Not only did DBeaver fail to dump a database, it failed to IMPORT a database and it failed on commented lines. For example, this line caused an error

`-- MySQL dump 10.13  Distrib 8.0.38, for Win64 (x86_64)`

That's right, a commented line made it fail.

I no longer need help with this issue because I don't intend to use this software. It's clearly not ready for actual users. Good luck with your little project

**Comment by ShadelessFox** [comment]
I'm thrilled to hear MySQL Workbench is meeting your needs. I wish you all the best with your superior choice and smooth sailing ahead.
